src/components/filtermenu/filtermenu.js

```diff
@@ -124,14 +124,15 @@ function saveValues(context, settings, settingsKey) {
     userSettings.setFilter(settingsKey + '-filter-VideoTypes', videoTypes.join(','));
 
     // Series status
-    const seriesStatuses = [];
-    elems = context.querySelectorAll('.chkSeriesStatus');
-
-    for (let i = 0, length = elems.length; i < length; i++) {
-        if (elems[i].checked) {
-            seriesStatuses.push(elems[i].getAttribute('data-filter'));
-        }
-    }
+    // TODO: Add filtering by series status?
+    // const seriesStatuses = [];
+    // elems = context.querySelectorAll('.chkSeriesStatus');
+
+    // for (let i = 0, length = elems.length; i < length; i++) {
+    //     if (elems[i].checked) {
+    //         seriesStatuses.push(elems[i].getAttribute('data-filter'));
+    //     }
+    // }
```

Comment:
Maybe just remove the code instead of leaving commented code?

Reply:
I commented it out because I suspected it would work with a minor fix... but I was being lazy and didn't want to test that at the time. Now I did and it works, so I just committed the fix here. :sweat_smile: